### ISSUE TITLE
is the schema.sql automatically being import in vercel?

### ISSUE THREAD

**Issue by goldcoders** [vercel/nextjs-subscription-payments#86]
I would like to know if when deploying in vercel and adding supabase integration will automatically
migrate the schema.sql to my supabase account

I would want to automate the process in deploying a website database with supabase with vercel
by abstracting it to the user,

If this is possible, then its awesome, i would migrate all my netlify sites to vercel for this reason :)

**Comment by thorwebdev** [comment]
Yes, the schema is automatically run via a Vercel deploy integration: https://vercel.com/docs/integrations

However, it is not automatically read from GitHub but rather we have an identifier for this application which then spins up the correct schema, so this is currently not generically available for any project.

Can you outline your intended workflow a little more?

**Comment by laky** [comment]
> However, it is not automatically read from GitHub but rather we have an identifier for this application which then spins up the correct schema, so this is currently not generically available for any project.

Do I read this correctly that making changes to the schema.sql will make no impact on what tables get created on deploy? What's the best way to add new tables etc then? Any recommendation?
